src/main/kotlin/xyz/crossward/entities/Entry.kt

```diff
@@ -10,8 +10,8 @@ import javax.persistence.*
         UniqueConstraint(columnNames = ["date", "user_id"])
     ]
 )
-class Entry(
+data class Entry(
     @Id @Column(name = "user_id") val userId: String,
-    @Column val date: LocalDate,
+    @Column val date: LocalDate? = null,
```

Comment:
Im not sure it matters on the Kotlin end, but SQLite doesnt actually have a Date datatype. The dates are entered into the database as strings in ISO format (YYYY-MM-DD)

Reply:
That said, sqlite allows us to specify the column as a Date in the schema